runtime/runtime/src/verifier.rs

```diff
@@ -326,6 +328,18 @@ pub(crate) fn validate_actions(
         });
     }
 
+    #[cfg(feature = "protocol_feature_tx_size_limit")]
+    {
+        let total_size =
+            total_size(actions).map_err(|_| ActionsValidationError::IntegerOverflow)? as u64;
+        if total_size > limit_config.max_transaction_size {
```

Comment:
Intuitively, it seems that `IntegerOverflow` should result in `TransactionSizeExceeded` instead of being a separate error?

Reply:
I agree, but in the code above `total_prepaid_gas` is handled in the same fashion.
@bowenwang1996 is the `IntegerOverflow` error important enough to track each case when it happens?

Reply:
In this case we can return `TransactionSizeExceeded`.

Reply:
Closing the thread because tx size computation was changed (see above).